README_EXAMPLES.md

```diff
@@ -37,8 +37,7 @@ MutableList<Person> people = Lists.mutable.with(
 ```
 Then we will `collect` their last names into a new `MutableList`, and finally output the names to a comma delimited String using `makeString`. 
 ```java
-MutableList<String> lastNames = people.collect(person -> person.getLastName());
-Assert.assertEquals("Smith, Watson, Williams", lastNames.makeString());
+Assert.assertEquals("Smith, Watson, Williams", lastNames.makeString(Person::getLastName, "", ",", ""));
```

Comment:
While this code works, it doesn't fit well with the example as it was meant to demonstrate `collect`. This could be a further refactoring of the example, after the lambda is converted to a method reference.

Reply:
I was thinking the same actually.. i'll revert this bit and will leave it as a later enhancement.